modules/tinymce/src/themes/silver/main/ts/api/Options.ts

```diff
@@ -301,7 +301,7 @@ const useElementPath = option('elementpath');
 const useBranding = option('branding');
 const getResize = option('resize');
 const getPasteAsText = option('paste_as_text');
-const getSidebarShow = option('sidebar_show');
+const getSidebarShow = wrapOptional(option('sidebar_show'));
```

Comment:
Why is this an `Optional` value now when we're immediately unwrapping it anyways?

Reply:
So I could use `Optional` helper functions to lowercase the input value. Realized that it is a not so good idea